src/ppk2_api/ppk2_api.py

```diff
@@ -472,7 +472,7 @@ def start_measuring(self):
 
     def stop_measuring(self):
         PPK2_API.stop_measuring(self)
-        PPK2_API.get_data(self) # flush the serial buffer (to prevent unicode error on next command)
+        PPK2_MP.get_data(self) # flush the serial buffer (to prevent unicode error on next command)
```

Comment:
I haven't tested this myself, but it is cleaner to do a `self.get_data` to use the instance of the class object instead of the class itself. Coult you give that a try?

Reply:
Yes, that makes alot more sense. Tested and it works. :)